hack/run_test

```diff
@@ -372,15 +376,19 @@ function run_master_restart_test() {
   local PASS=master
 
   echo "Testing failed master restart"
-  local cluster_args="-e POSTGRESQL_ADMIN_PASSWORD=pass -e POSTGRESQL_MASTER_USER=$PGUSER -e POSTGRESQL_MASTER_PASSWORD=$PASS"
   local cid_suffix="mrestart"
+  local network=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
```

Comment:
The thing with bash local variables is that their scope extends into all functions invoked in the function block they were declared in.
So the network variable is not really global, just globally accessible inside a specific test case. That is also why I have chosen to use lower case for its name, to avoid confusion (which happened anyway). Similarly for docker_args.